docs/source/create_channel/create_channel.md

```diff
@@ -0,0 +1,383 @@
+jq .data.data[0].payload.data.config config_block.json > config.json
+cp config.json config_copy.json
+```
+
+Add the Org2 peer that is joined to the channel as the anchor peer in the channel configuration:
+```
+jq '.channel_group.groups.Application.groups.Org2MSP.values += {"AnchorPeers":{"mod_policy": "Admins","value":{"anchor_peers": [{"host": "peer0.org2.example.com","port": 9051}]},"version": "0"}}' config_copy.json > modified_config.json
+```
+
+We can now convert both the original and updated channel configurations back into protobuf format and calculate the difference between them.
+```
+configtxlator proto_encode --input config.json --type common.Config --output config.pb
+configtxlator proto_encode --input modified_config.json --type common.Config --output modified_config.pb
+configtxlator compute_update --channel_id channel1 --original config.pb --updated modified_config.pb --output config_update.pb
+```
+
+Add the metadata back to the channel configuration update to create the new configuration block that we can use to update the channel:
+```
+configtxlator proto_decode --input config_update.pb --type common.ConfigUpdate --output config_update.json
+echo '{"payload":{"header":{"channel_header":{"channel_id":"channel1", "type":2}},"data":{"config_update":'$(cat config_update.json)'}}}' | jq . > config_update_in_envelope.json
+configtxlator proto_encode --input config_update_in_envelope.json --type common.Envelope --output config_update_in_envelope.pb
+```
+
+Navigate back to the `test-network` directory.
+```
+cd ..
+```
+
+Update the channel and set the Org2 anchor peer by issuing the following command:
+```
+peer channel update -f channel-artifacts/config_update_in_envelope.pb -c channel1 -o localhost:7050  --ordererTLSHostnameOverride orderer.example.com --tls --cafile ${PWD}/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/msp/tlscacerts/tlsca.example.com-cert.pem
+```
+
+You can confirm that the channel has been updated successfully by running the `peer channel info` command:
+```
+peer channel getinfo -c channel1
+```
+Now that the channel has been updated by adding two channel configuration blocks to the channel genesis block, the height of the channel will have grown to three:
+```
+Blockchain info: {"height":3,"currentBlockHash":"eBpwWKTNUgnXGpaY2ojF4xeP3bWdjlPHuxiPCTIMxTk=","previousBlockHash":"DpJ8Yvkg79XHXNfdgneDb0jjQlXLb/wxuNypbfHMjas="}
+```
+
+## Deploy a chaincode to the new channel
+
+We can confirm that the channel was created successfully by deploying a chaincode to the channel. We can use the `network.sh` script to deploy the Fabcar chaincode to any test network channel. Deploy a chaincode to our new channel using the following command:
+```
+./network.sh deployCC -c channel1
+```
+
+If successful, you should see the chaincode being deployed to the channel in your logs. The chaincode is invoked to add data to the channel ledger and then queried.
```

Comment:
When successful

Reply:
After you run the command?

Reply:
If successful -> When the command is successful